altair/vegalite/v5/api.py

```diff
@@ -3816,33 +3816,140 @@ def add_selection(self, *params) -> Self:  # noqa: ANN002
         return self.add_params(*params)
 
     def interactive(
-        self, name: str | None = None, bind_x: bool = True, bind_y: bool = True
+        self,
+        name: Optional[str] | None = None,
```

Comment:
`name` doesn't seem to be in-use yet, but unless you're planning to have different behaviour for `Undefined` vs `None` this could be simplified to:

```suggestion
        name: str | None = None,
```

When you wrote this, the it would have referred to `typing.Optional` and not the `altair`-specific type:

https://github.com/vega/altair/blob/f0c1e0a77bf8e3660fdabf484229a47a021d1f8c/altair/utils/schemapi.py#L787-L817